homeworks/01/chat-system/src/homework/TCPClient.java

```diff
@@ -0,0 +1,44 @@
+package homework;
+
+import java.io.*;
+import java.net.*;
+import java.util.Scanner;
+
+class TCPClient {
+    public static void main(String argv[]) throws Exception {
+        String command;
+        String nickname = "Guest";
+
+        Scanner scanner = new Scanner(new InputStreamReader(System.in));
+        command = scanner.nextLine();

```

Comment:
Currently the received messages aren't printed immediately. To fix that you could start another thread who's job would be to constantly listen for new messages and print them.